pulse_actions/handlers/route_functions.py

```diff
@@ -0,0 +1,14 @@
+import pulse_actions.handlers.treeherder_buildbot as treeherder_buildbot
+import pulse_actions.handlers.treeherder_resultset as treeherder_resultset
+import logging
+
+logging.basicConfig(format='%(levelname)s:\t %(message)s')
+LOG = logging.getLogger()
+
+def route(data, message, dry_run):
+    if data["action"] == "backfill":
+        treeherder_buildbot.on_buildbot_event(data, message, dry_run)
+    elif data["action"] == "trigger_missing_jobs" or data["action"] == "trigger_all_talos_jobs":
+        treeherder_resultset.on_resultset_action_event(data, message, dry_run)
+    else:
+        LOG.error("We currently only support 'backfill', 'trigger_missing_jobs', 'trigger_all_talos_jobs' for multiple listeners.")
```

Comment:
I like the error message, but for exchanges that don't have "action" as a key it will not show up correctly (we would get a KeyError, but this is not really a problem right now because the only topic base that doesn't have a 'action' ). That can be prevented by using:
```
action = data.get('action')
```
instead of data["action"]